### ISSUE TITLE
qt forces OpenSSL:no_zlib

### ISSUE THREAD

**Issue by barcharcraz** [bincrafters/community#587]
### Description of Problem, Request, or Question
Qt/5.12.0@bincrafters/stable forces OpenSSL:no_zlib=True, this means you can't use bincrafters qt with packages that require openssl with zlib support. cpprestsdk is an example of such a package (in the default configuration).

The following is a conanfile that runs into this problem.

```
[requires]
cpprestsdk/2.10.2@bincrafters/stable
qt/5.12.0@bincrafters/stable
OpenSSL/1.1.1a@conan/stable
[generators]
cmake
[options]
qt:shared=True
qt:openssl=True
qt:qtbase=True
OpenSSL:no_zlib=False
```

**Comment by ericLemanissier** [comment]
Could you try with https://github.com/bincrafters/conan-qt/tree/zlib if it suits your need ?

**Comment by barcharcraz** [comment]
is there a stream for that or do I need to clone and export it?

**Comment by ericLemanissier** [comment]
The recipe is not uploaded on a conan remote, so you would have to clone the repo and export it.

**Comment by ericLemanissier** [comment]
with https://github.com/bincrafters/conan-qt/commit/6a2891af89bb878c1bbe59503a7615932221c655 the qt recipe now forces OpenSSL:no_zlib=True to false, because qt recipe now requires zlib. The fix is available in qt/5.12.0@bincrafters/stable
